website/docs: Add Kubernetes Bootstrap Instructions (#9541)
goauthentik/authentik: website/docs/installation/automated-install.md

:::

Set the email address for the default `akadmin` user.

## Kubernetes

> Set the `akadmin`user password and token:

Reviewer: Can you state exactly where they set this? I am assuming in their `.env` file, but let's make it explicit in the instructions. ;-)

Reviewer 2: ```suggestion
In the Helm values, set the `akadmin`user password and token:
```

```
authentik:
  bootstrap_token: test
  bootstrap_password: test
```

> To store it in a secret, use:

Reviewer: ```suggestion
To store the password and token in a secret, use:
```

```

> envFrom

Reviewer 2: This should be under a `global:` key.

Also:
```suggestion
envFrom:
```

 - secretRef:

>      name: some-secret

Reviewer: ```suggestion
     name: _some-secret_
```

```

> where some-secret would contain the environment variables as in the documentation.

Reviewer: ```suggestion
where _some-secret_ contains the environment variables as in the documentation above.
```

Reviewer: Hi @Zapfmeister first off all, thank you very much for adding this to our docs; we definitely need more How To/procedural docs. This is a great addition. 

For the last sentence (line 42), can you explain a bit more what that means? Maybe give an example with the exact syntax? Which one of the three variables listed above in the docs would one use for the `some-secret`? Any of them? 

Thanks, I am still learning authentik myself, and I want to make the docs easy for newbies, too.
